Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>resources/*\nnode_modules/*\n.history/\n.idea/\nstatic/tsparticles\n.hugo_build.lock\nthemes\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 9136657deab1207855513bb4d6be99f26ed0f4bb)
+++ b/.gitignore	(date 1697697491046)
@@ -5,3 +5,5 @@
 static/tsparticles
 .hugo_build.lock
 themes
+assets/images/author/test*
+assets/images/author/ornob*
