Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>resources/*\nnode_modules/*\n.history/\n.idea/\nstatic/tsparticles\n.hugo_build.lock\nthemes\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 2d04eb8c18320a11eedff05cb2f2ee51a4850eaf)
+++ b/.gitignore	(date 1697713686083)
@@ -4,4 +4,6 @@
 .idea/
 static/tsparticles
 .hugo_build.lock
-themes
+assets/images/author/test*
+assets/images/author/ornob*
+go.sum
\ No newline at end of file
