Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>resources/*\nnode_modules/*\n.history/\n.idea/\nstatic/tsparticles\n.hugo_build.lock\nthemes\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 8dd9a17f05cb8abe948e301894d99606166b4551)
+++ b/.gitignore	(date 1697798826784)
@@ -4,4 +4,6 @@
 .idea/
 static/tsparticles
 .hugo_build.lock
-themes
+assets/images/author/test*
+assets/images/author/ornob*
+go.sum
\ No newline at end of file
